src/components/Nav.jsx

```diff
@@ -1,24 +1,28 @@
 /* eslint-disable */
 import React from "react";
-import { Link } from "react-router-dom";
+// import { Link } from "react-router-dom";
```

Comment:
- [OPTIONAL] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻